server/controller/FlightPhotoController.js

```diff
@@ -158,11 +158,10 @@ router.put(
   "/:id",
   authToken,
   checkParamIsUuid("id"),
-  checkStringObject("description"),
+  checkStringObjectNoEscaping("description"),
```

Comment:
Wie hatte sich das geäußert?

Reply:
Anführungszeichen -> wurden zu &quote, aber dafür gab's ja wieder das "unescape" in FlightService. Das ist aber jetzt dort rausgeflogen.